src/lib/themes.js

```diff
@@ -38,21 +47,21 @@ export const normalizeTheme = (data = {}) => {
   const theme = {
     colors: normalizeThemeColors(data.colors, defaultTheme.colors),
     images: {
-      additional_backgrounds: []
+      additional_backgrounds: [ "" ]
     }
   };
   const images = data.images ? data.images : {};
   if (images.headerURL) {
-    theme.images.additional_backgrounds = [ images.headerURL ];
+    theme.images.additional_backgrounds = [ normalizeThemeBackground(images.headerURL) ];
   }
   if (images.additional_backgrounds) {
-    theme.images.additional_backgrounds = images.additional_backgrounds;
+    theme.images.additional_backgrounds = [
+      normalizeThemeBackground(images.additional_backgrounds[0])
```

Comment:
So even though `additional_backgrounds` is an array, it will always have a maximum of 1 element, correct?
Is that because of a limitation in the front end UI? Or, something more interesting?

Reply:
The theme API can layer multiple backgrounds, but we're only using one in this project for now